.github/workflows/main.yml

```diff
@@ -0,0 +1,115 @@
+name: 🚀 CI/CD for Structurize
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    branches: [master]
+
+env:
+  node_version: 10.14
```

Comment:
@kirill-panoply please update the version to node 14

Reply:
I don't think we should do it since we're using this package in applications which are running on node 10.14

Reply:
as I mentioned in overflow PR, @OmriGLevy is working on node 14 migration

Reply:
Yeah so let's update node version here after it will be done.